operator/v1/types_csi_cluster_driver.go

```diff
@@ -61,6 +73,16 @@ const (
 // ClusterCSIDriverSpec is the desired behavior of CSI driver operator
 type ClusterCSIDriverSpec struct {
 	OperatorSpec `json:",inline"`
+	// StorageClassState determines if CSI operator should create and sync storage classes.
+	// If this field value is nil or Managed - CSI operator will continously reconcile
+	// storage class and create if necessary.
+	// If this field value is Unmanaged - CSI operator will not reconcile any previously created
+	// storage class.
+	// If this field value is Removed - CSI operator will delete the storage class it created previously.
+	// Defaults to nil
+	// +optional
+	// +kubebuilder:default=nil
+	StorageClassState *StorageClassStateName `json:"storageClassState,omitempty"`
```

Comment:
We don't allow pointers in general in configuration APIs, it makes it hard for end users to discover the options they might be able to configure. Instead just allow the empty string as a value and handle that as your `nil` case